dune-project

```
@@ -1,26 +1,28 @@
 (lang dune 3.0)
 
 (using menhir 2.1)
-
+(using dune_site 0.1)
 (using mdx 0.2)
+(using directory-targets 0.1)
 
 (cram enable)
 
+(generate_opam_files true)
+
 (name owi)
 
 (license AGPL-3.0-or-later)
 
 (authors
  "Léo Andrès <contact@ndrs.fr>"
- "Pierre Chambart <pierre.chambart@ocamlpro.com>")
+ "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+ "Filipe Marques")
```

Well thank you so much for the consideration! :smiley: You can just commit this suggestion if you'd like:

```suggestion
 "Filipe Marques <filipe.s.marques@tecnico.ulisboa.pt>")
```